### PR TITLE
frontend autofilter conversion report

### PR DESCRIPTION
**Pull request by yoouung**
- implement autofilter conversion report layout
- modify api request for conversion report data

p.s. dup module is not applied to this report

### CODE REVIEW

**Comment by yoouung** [comment]
### conversion report
image

File: dt-demo-be/app_lab.py [hunk 1]
```diff
@@ -114,5 +117,16 @@ def auto_file_list():
 
     return jsonify(user_uploaded_file_list)
 
+@app.route("/api/autofilter/convreport", methods=["GET"])
+def auto_get_conv_report():
+    app.logger.info("API request to get conversion report")
```
Comment (doorooful) [inline dt-demo-be/app_lab.py:122]: How about mentioning autofilter on log here?

File: dt-demo-be/app_lab.py [hunk 2]
```diff
@@ -114,5 +117,16 @@ def auto_file_list():
...
+    doc_list[0].run()
+    db_conn = dbconn.connect_to_db()
+
+    doc_list[0].fetchDetReportInfo(db_conn)
+    doc_list[0].fetchConvReportInfo(db_conn, "word")
```
Comment (doorooful) [inline dt-demo-be/app_lab.py:127]: It'd be better to add comment here!
``` sample.py
# Conversion option is automatically set as 'word'
```
Or how about add overloading `fetchConvReportInfo()` without option?

**Review by doorooful — APPROVED** [review]
Thank you 😀

**Comment by doorooful** [comment]
> ### conversion report
> image

Can you add autofilter page routing result as video?

**Comment by yoouung** [comment]
Uploading Screen Recording 2023-10-15 at 5.21.02 PM-1.mov…

**Comment by yoouung** [comment]
https://github.com/doorooful/CapstoneProject/assets/78146904/98d0b617-4137-47e0-9294-afbdb046fc3d

**Comment by yoouung** [comment]
https://github.com/doorooful/CapstoneProject/assets/78146904/2341800c-3250-4b6f-b190-90504c857b4c

**Comment by yoouung** [comment]
> > ### conversion report
> > image
> 
> Can you add autofilter page routing result as video?

https://github.com/doorooful/CapstoneProject/assets/78146904/6a602d16-20cb-49bf-8579-4c9f25e12855

**Pull request merged**